src/components/Search.tsx

```
@@ -0,0 +1,91 @@
+    }
+
+    document.addEventListener("keydown", down)
+    return () => document.removeEventListener("keydown", down)
+  }, [open, onOpenChange])
+
+  const filteredProducts = useCallback(() => {
+    if (!productsData) return []
+    return productsData.filter(
+      (product) =>
+        product.title &&
+        product.title.toLowerCase().includes(searchQuery.toLowerCase())
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)